src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
```

Comment:
Correct me if I am wrong, but it looks like you've not tested this method in `Main` class

Reply:
fixed

Reply:
fixed